app/views/shared/_address_lookup.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= page_template(
+      page_title: t(".heading", location:),
+      template: :basic,
+      form:,
+    ) do %>
+
+  <%= form.govuk_fieldset legend: { size: "xl", tag: "h1", text: t(".heading", location:) }  do %>
+    <%= form.govuk_text_field :postcode, label: { text: t(".label.postcode"), tag: "h2", size: "m" }, width: 10 %>
+    <%= form.govuk_text_field :building_number_name, label: { text: t(".label.building_number_name"), tag: "h2", size: "m" }, width: 10 %>
+    <%= govuk_warning_text(text: t(".helper_text")) %>
```

Comment:
Have we confirmed if this message will be the same for both correspondence and home address?